src/openms/source/ANALYSIS/QUANTITATION/IsobaricIsotopeCorrector.cpp

```diff
@@ -40,16 +39,15 @@ namespace OpenMS
 
     Matrix<double> correction_matrix = quant_method->getIsotopeCorrectionMatrix();
 
-    if (matrixIsIdentityMatrix(correction_matrix))
+    if (correction_matrix.getEigenMatrix().isIdentity(0.0))
```

Comment:
now works with eigen methods...
strangely this now throws for TMT11Plex (need to check why this is an identity matrix...)

Reply:
Maybe float equality? No idea.